### PR TITLE
Enable parallel heartbeat

### PR DESCRIPTION
**Pull request by troibe**
- Fixed typo
- To enable the tests again I ran them locally and recorded the time it took:
  - test_py 72s
  - test_sim 3s
  - test_cli 40s
  - test_parallel_all_serial 140s
  - test_parallel_steps 122s
  - test_parallel_flows 72s

Then I adjusted the timeouts.
Let's see if the CI is now happy:
[Daily CI run](https://github.com/siliconcompiler/siliconcompiler/actions/runs/5534007110)

### CODE REVIEW

**Comment by troibe** [comment]
This now passed and fairly sure those time limits are generous enough.

File: tests/examples/test_heartbeat.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def test_py(setup_example_test):
 
 @pytest.mark.eda
 @pytest.mark.quick
-@pytest.mark.timeout(300)
+@pytest.mark.timeout(15)
```
Comment (gadfort) [inline tests/examples/test_heartbeat.py:19]: you can remove this timeout the default is 120s

**Review by gadfort — CHANGES_REQUESTED** [review]
thanks. just remove the 15s timeout

File: tests/examples/test_heartbeat.py (hunk 1 above)
Comment (troibe) [reply tests/examples/test_heartbeat.py]: done

**Pull request merged**